reqwest-tracing/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+# [0.3.0] - 2022-06-10
+### Breaking
+- Created `RequestOtelSpanBackend` trait with `reqwest_otel_span` macro to provide extendable default request otel fields
 ### Fixed
```

Comment:
Was this released? Otherwise we need to cut a new patch before breaking.

Reply:
Let's cut a release in a separate PR and we can merge this after that once it's ready.